### PR TITLE
Doc: update KnownBugs and faq.

### PR DESCRIPTION
**Pull request by chainhelen**
1. Remove bug about unsupport systems, because we has pretty error msg.
2. Add usage for remote debugging.

Relate: #1883, #1829, #861, #1332.

### CODE REVIEW

File: Documentation/faq.md [hunk 1]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
 1. Assign the process its own TTY. This can be done on UNIX systems via the `--tty` flag for the 
 `dlv debug` and `dlv exec` commands. For the best experience, you should create your own PTY and 
 assign it as the TTY. This can be done via [ptyme](https://github.com/derekparker/ptyme).
+
+#### How can I use Delve for remote debugging?
+
+It is best not to use remote debugging on public network.
```
Comment (aarzilli) [inline Documentation/faq.md:59]: *on a public network
(also remove newline)

File: Documentation/faq.md [hunk 2]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
 1. Assign the process its own TTY. This can be done on UNIX systems via the `--tty` flag for the 
 `dlv debug` and `dlv exec` commands. For the best experience, you should create your own PTY and 
 assign it as the TTY. This can be done via [ptyme](https://github.com/derekparker/ptyme).
+
+#### How can I use Delve for remote debugging?
+
+It is best not to use remote debugging on public network.   
+If you have to do this, we recommend the use of ssh tunnels (or vpn network).
```
Comment (aarzilli) [inline Documentation/faq.md:60]: *using ssh tunnels or a vpn connection

File: Documentation/faq.md [hunk 3]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
 1. Assign the process its own TTY. This can be done on UNIX systems via the `--tty` flag for the 
 `dlv debug` and `dlv exec` commands. For the best experience, you should create your own PTY and 
 assign it as the TTY. This can be done via [ptyme](https://github.com/derekparker/ptyme).
+
+#### How can I use Delve for remote debugging?
+
+It is best not to use remote debugging on public network.   
+If you have to do this, we recommend the use of ssh tunnels (or vpn network).  
+
+Example for ssh tunnels.
```
Comment (aarzilli) [inline Documentation/faq.md:62]: ```##### Example```` instead

File: Documentation/faq.md [hunk 4]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
...
 assign it as the TTY. This can be done via [ptyme](https://github.com/derekparker/ptyme).
+
+#### How can I use Delve for remote debugging?
+
+It is best not to use remote debugging on public network.   
+If you have to do this, we recommend the use of ssh tunnels (or vpn network).  
+
+Example for ssh tunnels.  
+
+On remote server:
```
Comment (aarzilli) [inline Documentation/faq.md:64]: *Remote server:

File: Documentation/faq.md [hunk 5]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
...
+```
+dlv exec --headless --listen localhost:4040 /path/to/executable
+```
+
+On local client:
```
Comment (aarzilli) [inline Documentation/faq.md:69]: *Local client:

File: Documentation/faq.md [hunk 6]
```diff
@@ -53,3 +53,27 @@ the terminal TTY.
...
+1. enable forward
```
Comment (aarzilli) [inline Documentation/faq.md:70]: *connect to the server and start a local port forward

File: Documentation/faq.md [hunk 7]
```diff
@@ -53,3 +53,26 @@ the terminal TTY.
 1. Assign the process its own TTY. This can be done on UNIX systems via the `--tty` flag for the 
 `dlv debug` and `dlv exec` commands. For the best experience, you should create your own PTY and 
 assign it as the TTY. This can be done via [ptyme](https://github.com/derekparker/ptyme).
+
+#### How can I use Delve for remote debugging?
+
+It is best not to use remote debugging on a public network. If you have to do this, using ssh tunnels or a vpn connection.
```
Comment (aarzilli) [inline Documentation/faq.md:59]: s/If you have to do this, using/If you have to do this, we recommend using/

**Comment by chainhelen** [comment]
Done

**Review by aarzilli — APPROVED** [review]

**Pull request merged**